dist/src/main/config/broker.standalone.yaml.template

```
@@ -994,10 +994,10 @@
         # Configures if the RocksDB SST files should be partitioned based on some virtual column families.
         # By default RocksDB will not partition the SST files, which might have influence on the compacting of certain key ranges.
         # Enabling this option gives RocksDB some good hints how to improve compaction and reduce the write amplification.
-        # Benchmarks have show impressive results allowing to sustain performance on larger states, but it is not yet 100% clear what implications it else has except
-        # increasing the file count of runtime and snapshots.
+        # Benchmarks have show impressive results allowing to sustain performance on larger states.  This setting might
```

🔧 that's actually for certain?
```suggestion
        # Benchmarks have show impressive results allowing to sustain performance on larger states.  This setting will
```